data/virt_autotest/virt_logs_collector.sh

```diff
@@ -344,6 +344,51 @@ function collect_extra_logs_from_host() {
 	return ${ret_result}
 }
 
+#This function supports collecting guest configure xml file.
+#Typical usage: collect_guest_config logs_folder guest
+function collect_guest_config() {
+        local logs_folder=$1
+        local guest_name=$2
+        local guest_config="${logs_folder}/${guest_name}"
+
+        #Collect guest configure xml file
+        echo -e "Collect ${guest_name} configure xml file"
+        virsh dumpxml ${guest_name} | tee ${guest_config}-config.xml
+        if [[ $? -eq 0 ]];then
+           return 0
+        else
+           return 1
+        fi
+}
+
+#This function supports collecting and validating guest autoyast file.
+#Typical usage: collect_guest_autoyast logs_folder guest
+function collect_guest_autoyast() {
+        local logs_folder=$1
+        local guest_name=$2
+        local autoyast_dir="/srv/www/htdocs/install/autoyast"
+        local guest_autoyast="${autoyast_dir}/${guest_name}"
```

Comment:
@guoxuguang  If I remember correctly, the autoyast file in /srv/www/htdocs/install/autoyast only exists during guest installation, after the ctcs2 wrapped  script(calls virt-install.sh/vm-install.sh) finishes, the autoyast files under it will be cleared? The virt_logs_collector.sh will be called during openqa post_fail_hook which runs after ctcs2 script finishes, then it will not be possible to collect autoyast files. @Julie-CAO Please help double confirm.

Reply:
> @guoxuguang If I remember correctly, the autoyast file in /srv/www/htdocs/install/autoyast only exists during guest installation, after the ctcs2 wrapped script(calls virt-install.sh/vm-install.sh) finishes, the autoyast files under it will be cleared? The virt_logs_collector.sh will be called during openqa post_fail_hook which runs after ctcs2 script finishes, then it will not be possible to collect autoyast files. @Julie-CAO Please help double confirm.

Quick update,  had confirmed, after reproduced **guest installation failure**via ctcs2 wrapped script(calls vm-install.sh), guest autoyast file(s) was(were) still existed under /srv/www/htdocs/install/autoyast on our vm host.  

Moreover, had confirmed, after trigger guest installation via ctcs2 wrapped script(calls vm-install.sh) successfully, guest autoyast file(s) was(were) still existed under /srv/www/htdocs/install/autoyast on our vm host too.
So, found guest installation failure->call base class post_fail_hook()->call collect_host_and_guest_logs->call virt_logs_collector.sh->call collect_guest_autoyast() to collect guest autoyast file

Reply:
/srv/www/htdocs/install/autoyast/<vm_name>.xml is the exact autoyast control file which is used in guest installation. So I agree that it is the file we should upload.